### PR TITLE
SGP30 Air Quality Sensor Breakout

### PR DESCRIPTION
**Pull request by ckostovny**
adding component Adafruit SGP30 Air Quality Sensor Breakout - VOC and eCO2 - STEMMA QT / Qwiic
Product ID: 3709

### CODE REVIEW

File: components/i2c/sgp30/definition.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  "i2cAddresses": [ "0x58" ],
+  "subcomponents": [
+    {
+      "displayName": "eCO2",
+      "sensorType": "co2",
+      "defaultPeriod": 900
+    },
+    {
+      "displayName": "TVOC",
+      "sensorType": "unspecified",
```
Comment (brentru) [inline components/i2c/sgp30/definition.json:12]: @lorennorman  please see - we need to add "tvoc" (lcase) as a `sensorType`

Borrowing from the [CircuitPython Sensor Units](https://docs.circuitpython.org/en/latest/docs/design_guide.html#sensor-properties-and-units), let's call it "tvoc", it's a float and writes the "Total Volatile Organic Compounds in ppb".

**Comment by brentru** [comment]
@ckostovny  This looks great, thank you for the PR. 

TVOC is not currently a SI unit defined in Wippersnapper (this is our first sensor that sends TVOC data). I will pull this in and approve it once this sensor property is added by @lorennorman. Thank you for your patience.

**Comment by lorennorman** [comment]
@brentru I've added this sensor type to the validator for this repo and re-run the validator Action, which just passed!

It won't _actually_ work until we add it to the MQTT broker, which is waiting for it to be added to the protobufs, so I think we don't want to merge this until that part is ready.

**Comment by ckostovny** [comment]
I tried to look at the guide to add an i2c driver for the SGP30, but it looks like the Arduino library for the SGP30 isn't using the "Adafruit Unified Sensor Driver" and I don't understand C enough to proceed with adding an i2c driver.

**Pull request closed without merging**